robottelo/decorators/host.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import re
+import six
+import unittest2
+
+from robottelo import ssh
+from robottelo.config import settings
+from functools import wraps
+
+if six.PY3:
+    from functools import lru_cache
+elif six.PY2:
+    # workaround only for this module, its not doing exactly the same as
+    # original from Python 3, but it fits this specific case
+    def lru_cache(maxsize):
+        def decorator(func):
+            cache = []
+
+            @wraps(func)
+            def wrapper():
+                if not cache:
```

Comment:
first time cache=[] and func is executed making cache=['RHEL']. So next time "not cache" returns False. But once I would not want this method been executed more then one time, I double checked making a [test](https://github.com/SatelliteQE/robottelo/blob/master/tests/robottelo/decorators/test_host.py#L47). Once host version is aways the same, running ssh command would waste a lot of resources and time ;)